README.md

```diff
@@ -7,14 +7,170 @@
+      access to the database.
+* You have an Odoo v8 database named `odoo_v8`
+* You have an Odoo v8 filesystem on your host at `/var/lib/odoo_v8`
+* You want to upgrade to Odoo v10
+
+You would run the following commands:
+
+```bash
+export PG_USER=odoo_user
+export PG_PASSWORD=odoo_password
+
+# Migrate v8 to v9
+docker run \
+    --link postgresql:db \
+    --link postgresql:db_old \
+    -v /var/lib/odoo_v8:/var/lib/odoo_old \
+    -v /var/lib/odoo_v9:/var/lib/odoo \
+    -e "PGDATABASE=odoo_v9" \
+    -e "PGUSER=${PG_USER}" \
+    -e "PGPASSWORD=${PG_PASSWORD}" \
+    -e "DB_SOURCE=odoo_v8" \
+    -e "DB_TARGET=odoo_v9" \
+    laslabs/odoo-upgrade:9.0
+# Migrate v9 to v10
+docker run \
+    --link postgresql:db \
+    --link postgresql:db_old \
+    -v /var/lib/odoo_v9:/var/lib/odoo_old \
+    -v /var/lib/odoo_v10:/var/lib/odoo \
+    -e "PGDATABASE=odoo_v10" \
+    -e "PGUSER=${PG_USER}" \
+    -e "PGPASSWORD=${PG_PASSWORD}" \
+    -e "DB_SOURCE=odoo_v9" \
+    -e "DB_TARGET=odoo_v10" \
+    laslabs/odoo-upgrade:10.0
+```
+
+Running the above, you will be left with an Odoo v9 and an Odoo v10 database &
+filesystem. They will be named `odoo_v9` and `odoo_v10` respectively.
+
+You typically want to run one migration at a time, and validate that it works before
+proceeding to the next one. Living on the edge can be fun though.
+
+Docker-Compose Instructions
+===========================
+
+The above section is mostly instructional usage, in that it uses plain Docker and
+does not consider custom addons that you may have in your installation. Instead, you
+would want to use a `docker-compose.yml` file similar to the below:
+
+```yml
+version: "2"
+
+services:
+
+  postgresql:
+    image: postgres:9.6-alpine
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_PASSWORD: odoo_password
+      POSTGRES_USER: odoo_user
+    volumes:
+    - ~/postgresql_data/:/var/lib/postgresql/data/
+
+  # This is your original Odoo v8 deploy.
+  # Use this as a reference for changes that need to be made in other sections
+  # in order to match your actual deploy. An environment file makes this easier,
+  # but was excluded in the interest of a one file example.
+  #
+  # odoo_v8:
+  #  image: your_organization/your_odoo_scaffold_image:8.0
+  #  environment:
+  #    - PGDATABASE: odoo_v8
+  #    - PGPASSWORD: odoo_password
+  #    - PGUSER: odoo_user
+  #  volumes:
+  #    - ~/odoo_v8_data:/var/lib/odoo
+  #  links:
+  #    - postgresql:db
+
+  openupgrade_v9:
+    image: laslabs/odoo-upgrade:9.0
+    volumes:
+      - ~/odoo_v8_data:/var/lib/odoo_old
+      - ~/odoo_v9_data:/var/lib/odoo
+    environment:
+      - DB_SOURCE: odoo_v8
+      - DB_TARGET: odoo_v9
+      - PGDATABASE: odoo_v9
+      - PGUSER: odoo_user
+      - PGPASSWORD: odoo_password
+    links:
+      - postgresql:db
+      - postgresql:db_old
+
+  custom_upgrader:
+    image: your_organization/your_odoo_scaffold_image:9.0
+    command: "autoupdate"
```

Comment:
This should only work when having installed `module_auto_update`, and when that addon is already migrated to the new odoo version, right?

Reply:
Ahhh crap good call. I'll switch this to a standard `odoo --update=all --workers=0` then.

I also submitted Tecnativa/docker-odoo-base#113 to add a note of such